src/App.css

```diff
@@ -16,17 +16,15 @@
 .canvas {
   background-color: var(--element-background);
   position: relative;
-  height: 60vmin;
-  width: 60vmin;
-  min-height: 60vmin;
-  min-width: 60vmin;
+  height: 70vmin;
+  width: 70vmin;
 }
 
 .node {
   height: 2vh;
   width: 2vh;
   border-radius: 50%;
-  font-size: 2vh;
+  font-size: 1.65vh;
```

Comment:
Font sizes here should be relative to canvas size, so I would suggest `font-size: 1vmin` in `.canvas` and here `1.65em`